src/settings/ClosingReasons.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import ClosingReasonsForm from './ClosingReasonsForm';
+
+
+class ClosingReasons extends Component {
+  static manifest = Object.freeze({
+    recordId: {},
+    closingReasons: {
+      type: 'okapi',
+      records: 'configs',
+      path: 'configurations/entries?query=(module=ORDERS and configName=closing-reasons)',
```

Comment:
this is specific path for GET, please break it on path and params for GET, no need to use it herer and override for other methods to the default one